resources/assetstore/charts/minio/templates/deployment.yaml

```diff
@@ -37,15 +40,17 @@ spec:
   template:
     metadata:
       name: {{ template "minio.fullname" . }}
-      # MODIFIED: Block - START
-      {{- if .Values.deployment.annotations }}
-      annotations:
-{{ toYaml .Values.deployment.annotations | indent 8 }}
-      {{- end }}
-      # MODIFIED: Block - END
       labels:
         app: {{ template "minio.name" . }}
         release: {{ .Release.Name }}
+      {{- if .Values.podAnnotations }}
```

Comment:
Extract this condition to global? You also use this logic in statefulset.

Reply:
`values.yaml` may contain only values, logic is not supported.